contracts/honzon/Honzon.sol

```diff
@@ -0,0 +1,106 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+
+pragma solidity ^0.8.0;
+
+import "./IHonzon.sol";
+
+contract Honzon is IHonzon {
+    address constant private precompile = address(0x0000000000000000000000000000000000000409);
+
+    /**
+     * @dev Adjust CDP position
+     * Returns a boolean value indicating whether the operation succeeded.
+     */
+    function adjustLoan(address currencyId, int256 collateralAdjustment, int256 debitAdjustment)
+    public
+    override
+    returns (bool) {
+        require(collateralAdjustment != 0 && debitAdjustment != 0, "Honzon: adjustment amounts are zero");
+
+        (bool success, bytes memory returnData) = precompile.staticcall(abi.encodeWithSignature("adjustLoan(address,address,int256,int256)", msg.sender, currencyId, collateralAdjustment, debitAdjustment));
```

Comment:
@ferrell-code staticcall is for executing something without applying changes, like read or view. In this case it needs to be .call(